docs/templates.md

```diff
@@ -47,6 +47,9 @@ The following functions are defined.
   non-empty lines by the given `prefix`.
 * `label(label: Template, content: Template) -> Template`: Apply label to
   the content. The `label` is evaluated as a space-separated string.
+* `raw_escape_sequence(content: Template) -> Template`: Preserves any escape
+  sequences in `content` (i.e., bypasses sanitization) and strips formatting.
```

Comment:
nit: "strips formatting" is a bit confusing. Just say "strips labels" or "strips color labeles"?